applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes_fractional.h

```diff
@@ -0,0 +1,608 @@
+         * @param rRHS The effect of pressure discontinuity is implemented as an interfacial integral on the RHS
+         */
+        void SurfaceTension(
+            const double SurfaceTensionCoefficient,
+            const Vector &rCurvature,
+            const Vector &rInterfaceWeights,
+            const Matrix &rInterfaceShapeFunctions,
+            const std::vector<array_1d<double, 3>> &rInterfaceNormalsNeg,
+            VectorType &rRHS);
+
+        /**
+         * @brief Condense the enrichment
+         * This method performs the static condensation of the enrichment terms, by adding
+         * its local contributions to both the LHS and RHS elemental matrices.
+         * @param rData Element data container
+         * @param rLeftHandSideMatrix Reference to the element Left Hand Side matrix
+         * @param rRightHandSideVector Reference to the element Right Hand Side vector
+         * @param rVTot Common N-S equations term associated to pressure enrichment DOFs
+         * @param rHTot Pressure enrichment contribution related to velocity and pressure DOFs
+         * @param rKeeTot Pressure enrichment contribution related to pressure enrichment DOFs
+         * @param rRHSeeTot Right Hand Side vector associated to the pressure enrichment DOFs
+         */
+        void CondenseEnrichmentWithContinuity(
+            const TElementData &rData,
+            Matrix &rLeftHandSideMatrix,
+            VectorType &rRightHandSideVector,
+            const MatrixType &rVTot,
+            const MatrixType &rHTot,
+            MatrixType &rKeeTot,
+            const VectorType &rRHSeeTot);
+
+        /**
+         * @brief Condense the enrichment without penalty
+         * This method performs the static condensation of the enrichment terms, by adding
+         * its local contributions to both the LHS and RHS elemental matrices.
+         * Pressure continuity along cut edges is not penalized in this function
+         * Volume ratio is not checked in this function
+         * @param rLeftHandSideMatrix Reference to the element Left Hand Side matrix
+         * @param rRightHandSideVector Reference to the element Right Hand Side vector
+         * @param rVTot Common N-S equations term associated to pressure enrichment DOFs
+         * @param rHTot Pressure enrichment contribution related to velocity and pressure DOFs
+         * @param rKeeTot Pressure enrichment contribution related to pressure enrichment DOFs
+         * @param rRHSeeTot Right Hand Side vector associated to the pressure enrichment DOFs
+         */
+        void CondenseEnrichment(
+            Matrix &rLeftHandSideMatrix,
+            VectorType &rRightHandSideVector,
+            const MatrixType &rVTot,
+            const MatrixType &rHTot,
+            MatrixType &rKeeTot,
+            const VectorType &rRHSeeTot);
+
+        void AddSurfaceTensionContribution(
+            const TElementData &rData,
+            MatrixType &rInterfaceShapeFunction,
```

Comment:
I would do custom class/struct to reduce the number of arguments

Reply:
Indeed, I'd directly remove any surface tension related stuff as we are not using it in here.

Reply:
You're right, I'll remove everything related to dynamic droplets.